submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,73 @@
+         <a href="#">Book Rentals</a></h2>
+
+         <li><a href="#">New Releases</a></li><li> <a href="#">General Book Shelf</a></li><li> <a href="#">Today's Deals</a></li><li> 
+         <a href="#">Registration</a></li><li> <a href="#">Teachers Corner</a></li><li> <a href="#">Students Corner</a></li><li> 
+         <a href="#">Contact</a></li><li> <a href="#">Order Offline</a></li>
+  </header>
+   <main>
+    <div>
+     <li>
+      <div>
+       <h2><a href="">Cambridge Checkpoint Mathematics Coursebook 7</a></h2>
+	<img src="image/cambridge-checkpoint-mathematics-coursebook-7.jpg" alt="cambridge-checkpoint-mathematics-coursebook-7"/>
+         <h2>Price: $20.82</h2>
+          <h4>Customers Ratings: 5.0  (110 Reviews)</h4>
+          </div>
+           <div>
+           <h2>Product description</h2>
+             Written by well-respected authors, the Cambridge Checkpoint Mathematics suite provides a comprehensive structured resource 
+	     which covers the full Cambridge Secondary 1 Mathematics framework in three stages. This brightly illustrated Coursebook for 
+	     Stage 7 offers a comprehensive introduction to all topics covered in the syllabus. Worked examples show students how to tackle 
+	     different problems, and plenty of exercise questions prepare students for the different types of question they will face in their 
+	     exam. Coverage of the Problem Solving framework is integrated throughout the course, with questions relating to the framework 
+	     statements highlighted in the Coursebook and Practice Book.
+              </p>
+            </div>
+          </li>
+          <hr />
+          <li>
+            <div>
+              <h2><a href="">Cambridge Global English Stage 7 Coursebook with Audio CD</a></h2>
+		<img src="image/cambridge-global-english-stage-7-coursebook-with-audio-cd.jpg" alt="cambridge-global-english-stage-7-coursebook-with-audio-cd"/>
+             <h2>Price: $36.32</h2>
+             <h4>Customers Ratings: 5.0  (115 Reviews)</h4>
+     
+              <h2>Product description</h2>
+                Cambridge Global English Stages 7-9 follow the Cambridge Lower Secondary English as a Second Language (ESL) Curriculum Framework. 
+		Coursebook 7 is organised into eighteen thematic units of study based on the Cambridge International English Scheme of Work for 
+		Stage 7. The topics and situations in Cambridge Global English have been selected to reflect this diversity and encourage learning 
+		about each other's lives through the medium of English. It presents realistic listening, speaking, reading and writing tasks, as 
+		well as end-of-unit projects similar to those students might encounter in the context of a first-language school. After every 
+		other unit, there is a literature spread, featuring authentic texts from a variety of sources. CEFR Level - B1
+              </p>
+        <hr />
+           <p><h3>Customers who viewed this item also viewed</h3>
```

Comment:
the p tag here should have a closing tag at the end

Reply:
Is that all, please help me review all possible changes so that I won't keep repeating the processes. Thank you

Reply:
Where am I to add the closing tag to the main tag?